Packs/GreyNoise_Community/README.md

```diff
@@ -0,0 +1,14 @@
+GreyNoise tells security analysts what not to worry about. We do this by curating data on IPs that saturate security
+tools with noise. This unique perspective helps analysts confidently ignore irrelevant or harmless activity, creating
+more time to uncover and investigate true threats
+
+##### What does this pack do?
+This pack allows for analysts to integrate the free, GreyNoise Community API data into their playbooks and workflows.
+It gives them the ability to determine if an IPv4 address is something they need to worry about or is just
+Internet-Background noise.
+
+- Allows analysts to query an IP from GreyNoise and determine if it is Internet-Background Noise.
+- Allows analysts to query an IP from GreyNoise and determine if it belongs to a benign service or provider.
+
+
+_For more information, visit our [GreyNoise Developer Documentation](https://developer.greynoise.io)_
```

Comment:
@bradchiappetta Suggest add a link for them to get more information on your site on upgrading to the full API

Reply:
And differences between this API/pack and the full one.